Sources/Widgets/SVG/SVGLandmarkRepresentation/index.js

```diff
@@ -118,9 +133,16 @@ function defaultValues(initialValues) {
       ...initialValues.circleProps,
     },
     textProps: {
-      fill: 'white',
       ...initialValues.textProps,
     },
+    fontProperties: {
+      fontColor: 'white',
+      fontSize: 16,
+      ...initialValues.fontProperties,
+    },
+    strokeFontProperties: {
+      ...initialValues.strokeFontProperties,
+    },
```

Comment:
why bother defining strokeFontProperties if you do not have default values, it is already done line 128.